keyboards/binepad/knobx1/keyboard.json

```diff
@@ -0,0 +1,96 @@
+{
+    "manufacturer": "Binepad",
+    "keyboard_name": "KnobX1",
+    "maintainer": "binepad",
+    "bootloader": "stm32duino",
+    "bootloader_instructions": "Hold down the layer button (small button on the bottom left) and plug in the keyboard.",
+    "diode_direction": "COL2ROW",
+    "eeprom": {
+        "wear_leveling": {
+            "backing_size": 1024
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {
+                "pin_a": "B12",
+                "pin_b": "B13",
+                "resolution": 2
+            }
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "midi": true,
+        "mousekey": true,
+        "nkro": false,
```

Comment:
```suggestion
```

Reply:
Don't need NKRO for a 1 key keyboard.  But I get it.